packages/client/modules/userDashboard/components/UserSettingsForm/UserSettingsForm.tsx

```diff
@@ -0,0 +1,129 @@
+import useAtmosphere from '../../../../hooks/useAtmosphere'
```

Comment:
+1 It probably made sense to move the logic out, but it makes review harder if unrelated changes are intermingled. No need to revert this, but just a note for the future.

Reply:
If there are dependencies, one PR can target the feature branch of another PR instead of master. Once the first branch is merged, GitHub will change the target branch of the second one to master.

Reply:
cool, make sense! thanks @Dschoordsch